### PR TITLE
Added a lot of Oxford commas

### PR DESCRIPTION
**Pull request by anthonycrumley**
A regular expression was used to find a lot of missing Oxford
commas and add them.  The regular expression was as follows.

", ([a-zA-Z0-9.\`:'\"]+ ){1,6}(or|and) "

### CODE REVIEW

File: guides/source/debugging_rails_applications.md [hunk 1]
```diff
@@ -348,7 +348,7 @@ by asking the debugger for help. Type: `help`
   break      -- Sets breakpoints in the source code
   catch      -- Handles exception catchpoints
   condition  -- Sets conditions on breakpoints
-  continue   -- Runs until program ends, hits a breakpoint or reaches a line
+  continue   -- Runs until program ends, hits a breakpoint, or reaches a line
```
Comment (matthewd) [inline guides/source/debugging_rails_applications.md]: ✂️ 

We can't editorialize on byebug's output :smile:

File: guides/source/debugging_rails_applications.md [hunk 2]
```diff
@@ -365,7 +365,7 @@ by asking the debugger for help. Type: `help`
   irb        -- Starts an IRB session
   kill       -- Sends a signal to the current process
   list       -- Lists lines of source code
-  method     -- Shows methods of an object, class or module
+  method     -- Shows methods of an object, class, or module
```
Comment (matthewd) [inline guides/source/debugging_rails_applications.md]: ✂️

File: guides/source/debugging_rails_applications.md [hunk 3]
```diff
@@ -575,7 +575,7 @@ Let's have `byebug` help us with it.
   Shows variables and its values
 
 
-  var all      -- Shows local, global and instance variables of self.
+  var all      -- Shows local, global, and instance variables of self.
```
Comment (matthewd) [inline guides/source/debugging_rails_applications.md]: ✂️

**Review by matthewd — COMMENTED** [review]
Thanks!

**Comment by anthonycrumley** [comment]
@matthewd Thanks for the feedback.  I made changes as per your comments.

**Review by matthewd — APPROVED** [review]
Thanks again. If you could squash this down to a single commit, I think it's good to go :+1:

**Comment by anthonycrumley** [comment]
@matthewd This is now squashed down to one commit.  👍

**Comment by anthonycrumley** [comment]
@matthewd For code change responses to comments on a pull request, does the Rails team find it easier to review a separate commit that is later squashed or an amendment of the original commit?

Thanks for your feedback.  😄

**Comment by matthewd** [comment]
Personally, I'd say it depends whether the change is short enough, or has changed so much, that re-reading the full diff is the easiest way to review it -- in which case pre-squashing saves a back-and-forth. In a case like this, it was much easier for me to initially review as an additional commit. :+1:

**Pull request merged**